### PR TITLE
Bumps internal deps

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Changes:

CartoDB.js:
- https://github.com/CartoDB/cartodb.js/pull/1369
- https://github.com/CartoDB/cartodb.js/pull/1370
- https://github.com/CartoDB/cartodb.js/pull/1371

Deep-insights.js:
- https://github.com/CartoDB/deep-insights.js/pull/437

cc: @xavijam

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -156,6 +156,7 @@
     "test": "jasmine",
     "test-debug": "node-debug --no-preload ./node_modules/.bin/jasmine --",
     "test-watch": "watch 'npm test' $npm_package_config_npmJsDir",
+    "preupdate-internal-deps": "rm -rf node_modules/camshaft-reference node_modules/cartoassets node_modules/cartodb-deep-insights.js node_modules/cartodb.js",
```
Comment (xavijam) [inline package.json]: Important @CartoDB/frontend ^^^^

Comment (xavijam) [inline package.json]: BTW, why not adding that to the update-internal-deps "script"?

Comment (alonsogarciapablo) [inline package.json]: "preupdate-internal-deps" will be automagically executed every time we `npm run update-internal-deps` (it's a pre hook, more info [here](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/)).

Comment (xavijam) [inline package.json]: :+1:

**Pull request merged**